teamcity: failed tests on master: testrace/TestLogic, testrace/TestLogic/distsql, testrace/TestLogic/distsql/txn

The following tests appear to have failed:

[#289348](https://teamcity.cockroachdb.com/viewLog.html?buildId=289348):

```
--- FAIL: testrace/TestLogic (444.200s)
<autogenerated>:10: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic131842924
<autogenerated>:9: use -show-logs to present logs inline
--- FAIL: testrace/TestLogic/distsql (1.450s)
null
--- FAIL: testrace/TestLogic/distsql/txn (9.500s)
logic_test.go:1718: 
	testdata/logic_test/txn:624: expected "restart transaction: HandledRetryableTxnError: forced by crdb_internal.force_retry()", but found <nil>
logic_test.go:1737: testdata/logic_test/txn:627: too many errors encountered, skipping the rest of the input

```
Please assign, take a look and update the issue accordingly.

--- a/pkg/sql/logictest/testdata/logic_test/txn
+++ b/pkg/sql/logictest/testdata/logic_test/txn
@@ -622,14 +622,14 @@ statement ok
 BEGIN TRANSACTION; SAVEPOINT cockroach_restart; SELECT 1;
 
 query error pgcode 40001 restart transaction: HandledRetryableTxnError: forced by crdb_internal.force_retry()
-SELECT CRDB_INTERNAL.FORCE_RETRY('50ms':::INTERVAL)
+SELECT CRDB_INTERNAL.FORCE_RETRY('1h':::INTERVAL)
 
 statement ok
 ROLLBACK TO SAVEPOINT COCKROACH_RESTART;
 
 # This is the automatic retry we care about.
 statement ok
-SELECT CRDB_INTERNAL.FORCE_RETRY('100ms':::INTERVAL)
+SELECT CRDB_INTERNAL.FORCE_RETRY('50ms':::INTERVAL)
 
 statement ok
 ROLLBACK

